src/js/common/polyfills.js

```diff
@@ -0,0 +1,26 @@
+// Common Javascript environment setup to be used by all entrypoints.
+//
+// Keep this short. It should mostly be polyfills, global site style, and any
+// truly pervasive libraries. Be careful if you are tempted to put jquery or
+// other such libraries in here. Most of the site does not use these legacy
+// frameworks and it belongs in a lower-level module.
+
+require('babel-polyfill');
```

Comment:
Yeah, `babel-preset-env` has a transform that takes `require('babel-polyfill')` and turns into a list of require statements for the specific polyfills, sort of like this:

```js
require('core-js/es6/array/find');
require('core-js/es6/array/from');
require('core-js/es6/array/fill');
```

etc, based on what our supported browsers need.